resources/provider-head.txt

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// DO NOT EDIT THIS FILE MANUALLY! EDIT resources/provider-header.txt AND RUN scripts/codegen.sh!
```

Comment:
This references a file that was deleted.  I'm not sure that's what you want?

Reply:
I'd also probably change the wording to something like 'DO NOT EDIT PROVIDER.CPP directly!' instead of 'this file' since that's ambiguous if you're actually trying to edit the template file.